lib/iex/lib/iex/helpers.ex

```diff
@@ -34,6 +34,7 @@ defmodule IEx.Helpers do
     * `l/1`           - loads the given module's beam code
     * `ls/0`          - lists the contents of the current directory
     * `ls/1`          - lists the contents of the specified directory
+    * `nl/2`          - deploys local beam code to a list of connected nodes
```

Comment:
:scissors: "connected". The node does not need to be connected when the helper is called. If not connected it will try to connect.